kinder/ci/workflows/external-ca-tasks.yaml

```diff
@@ -0,0 +1,127 @@
+# IMPORTANT! this workflow is imported by external-ca-* workflows.
+version: 1
+summary: |
+  This workflow implements a sequence of tasks for testing the external-ca functionality.
+vars:
+  # vars defines default values for variable used by tasks in this workflow;
+  # those values might be overridden when importing this files.
+  kubernetesVersion: v1.16.0
+  baseImage: kindest/base:v20190403-1ebf15f
+  image: kindest/node:test
+  clusterName: kinder-external-ca
+  kubeadmVerbosity: 6
+tasks:
+  - name: pull-base-image
+    description: |
+      pulls kindest/base image with docker in docker and all the prerequisites necessary for running kind(er)
+    cmd: docker
+    args:
+      - pull
+      - "{{ .vars.baseImage }}"
+  - name: add-kubernetes-versions
+    description: |
+      creates a node-image-variant by adding Kubernetes version "kubernetesVersion"
+      to be used when executing "kinder do kubeadm-init"
+    cmd: kinder
+    args:
+      - build
+      - node-image-variant
+      - --base-image={{ .vars.baseImage }}
+      - --image={{ .vars.image }}
+      - --with-init-artifacts={{ .vars.kubernetesVersion }}
+      - --with-upgrade-artifacts={{ .vars.kubernetesVersion }}
+      - --loglevel=debug
+    timeout: 15m
+  - name: create-cluster
+    description: |
+      create a set of Nodes ready for hosting the Kubernetes cluster
+    cmd: kinder
+    args:
+      - create
+      - cluster
+      - --name={{ .vars.clusterName }}
+      - --image={{ .vars.image }}
+      - --control-plane-nodes=3
+      - --worker-nodes=2
+      - --loglevel=debug
+    timeout: 5m
+  - name: setup-external-ca
+    description: |
+      Prepares the nodes in the cluster with certificates and kubeconfig files so that the
+      cluster can start without the root ca.key file
+    cmd: kinder
+    args:
+      - do
+      - setup-external-ca
+      - --name={{ .vars.clusterName }}
+      - --loglevel=debug
+      - --kubeadm-verbosity={{ .vars.kubeadmVerbosity }}
+  - name: init
+    description: |
+      Initializes the Kubernetes cluster with version "kubernetesVersion"
+      by starting the bootstrap control-plane node
+    cmd: kinder
+    args:
+      - do
+      - kubeadm-init
+      - --name={{ .vars.clusterName }}
+      - --loglevel=debug
+      - --kubeadm-verbosity={{ .vars.kubeadmVerbosity }}
+    timeout: 5m
+  - name: join
+    description: |
+      Join the other nodes to the Kubernetes cluster
+    cmd: kinder
+    args:
+      - do
+      - kubeadm-join
+      - --copy-certs=skip # skip certificate copy since "setup-external-ca" manages that
+      - --name={{ .vars.clusterName }}
+      - --ignore-preflight-errors=all # also skip checks for existing kubelet.conf file
```

Comment:
might be worth passing something more specific than all? I won't be sure to not hide other problems by chance